fishjam/media_events/shared.proto

```diff
@@ -0,0 +1,27 @@
+syntax = "proto3";
+
+package fishjam.media_events;
+
+enum Encoding {
+  ENCODING_UNSPECIFIED = 0;
+  ENCODING_LOW = 1;
+  ENCODING_MEDIUM = 2;
+  ENCODING_HIGH = 3;
+}
+
+message Metadata {
+  string json = 1;
+}
+
+message MidToTrackId {
+  string mid = 1;
+  string track_id = 2;
+}
+
+// Contains information about an ICE candidate which will be sent to the peer/server
+message Candidate {
+  string candidate = 1;
+  int32 sdp_m_line_index = 2;
+  int32 sdp_mid = 3;
+  string username_fragment = 4;
+}
```

Comment:
In these two messages, `mid` is declared as both `int32` and `string`. Is it possible or valuable to declare it as either a `string` or an `int32`?